rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly-2025-05-05"
```

Comment:
This needs to match polars `rs-0.47` branch which currently is on `nightly-2025-04-19`
We also need to use the same nightly for CI/CD.